addons/test_mail/models/test_mail_models.py

```diff
@@ -30,6 +30,19 @@ def _notify_by_email_get_headers(self):
         return headers
 
 
+class MailTestSimpleUnfollow(models.Model):
+    """ A very simple model only inheriting from mail.thread when only
+    communication history is necessary
+    + allowing partner even without user to unfollow """
```

Comment:
"...communication history is necessary with automatic unfollow link in notification emails addition" ?

Reply:
yes good idea. I have slightly changed it because unfollow link is enabled by default for internal user and this flag enable it also for non internal user → “...communication history is necessary with unfollow link enabled in notification emails even for non-internal user.”